app/src/themes/directus-dark-2022.ts

```diff
@@ -0,0 +1,93 @@
+import { Theme } from '@directus/shared/types';
+
+export const darkTheme: Theme = {
+	name: 'Directus Dark 2022',
```

Comment:
>  in case we decide to modify the base Directus theme down the line. Having them separate would allow using the 'legacy' theme.

Seeing that there's some users, albeit not many, that wanted to go back to the "legacy" dark theme during the recent color palette update, I do very much agree with this point and don't see any harm with it. So I'm not opposed to leaving it as it is 👍